subprojects/packagefiles/boost-pfr/meson.build

```diff
@@ -0,0 +1,9 @@
+project(
+  'boost-pfr',
+  'cpp',
+  version: '1.88.0',
+  license: 'BSL-1.0',
+  default_options: ['cpp_std=c++11'],
+)
+
+boost_pfr_dep = declare_dependency(include_directories: 'include')
```

Comment:
Since the minimum Meson version permits it, there should be a `meson.override_dependency()` added here, and the wrap file `provide` line should be `dependency_names = boost-pfr`.